lib/ContractInteract/TokenHolder.js

```diff
@@ -0,0 +1,185 @@
+'use strict';
+
```

Comment:
Add getter for `token address`, `sessionWindow`, `sessionKeys`, `tokenRules`, `owner address`.

Reply:
I will add this in the ticket as enhancements.
even if we add these methods not sure if they will be used for now.